src/Main.elm

```diff
@@ -3,77 +3,432 @@ module Main exposing (main)
 import Browser
 import Data
 import Html exposing (Html)
-import Html.Attributes as Attrs
+import Json.Decode
+import Element as E exposing (Element)
+import Element.Font as F
+import Ui.Theme as T
+import Ui
+import Validate
 
 
+{-
 
----- MODEL ----
+TODO
+
+- Debounce so errors show up after finished typing.
+- Arrange validations errors such that submit button doesn't jump.
+- Highlight invalid input fields and add Aria.invalid tags.
```

Comment:
Oi!
Are we even adding Aria.invalid in our own UI components!? O_O
*runs away to check*